.gitignore

```diff
@@ -10,6 +10,7 @@
 /integration_tests/transform/*/coverage
 /integration_tests/transform/*/node_modules
 /integration_tests/*/node_modules
+/integration_tests/json_reporter/sum.result.json
```

Comment:
Instead of adding this file here, can we add a cleanup using `afterAll` in the test file to delete this file? Otherwise the test will not be so useful if the file already existed beforehand.